module2/success/learning_goals.md

```diff
@@ -14,36 +14,34 @@ The expected mastery level can be understood with the following scale:
 
 # Mastery
```

Comment:
Does everything in the Mastery section accurately represent what a student needs to know coming out of Mod 2?

Reply:
Yea, I think that covers it.